plugin/storage/badger/docs/storage-file-non-root-permission.md

```diff
@@ -0,0 +1,46 @@
+# Badger file permissions as non-root service
+
+After the release of 1.50, Jaeger's Docker image is no longer running with root privileges (in [#4783](https://github.com/jaegertracing/jaeger/pull/4783)), and issues such as "permission denied" occur when writing.
```

Comment:
```suggestion
After the release of 1.50, Jaeger's Docker image is no longer running with root privileges (in [#4783](https://github.com/jaegertracing/jaeger/pull/4783)). In some installations it may cause issues such as "permission denied" errors when writing data.
```